docs/concepts/comparison.rst

```diff
@@ -33,19 +31,17 @@ Similar Libraries
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
 | Supports multiple environments / backends | Yes                      | Yes                                  | No                         |
```

Comment:
What does this even mean?

Reply:
For example, both `j5` and `robot-api` can support multiple backends, such as a simulated environment. For `robot-api` this requires a separate `robotd` implementation.